src/components/Create/useCreateLink.tsx

```
@@ -608,49 +594,77 @@ export const useCreateLink = () => {
+                            )
+                        )
+                        await checkUserHasEnoughBalance({
+                            tokenValue: String(Number(tokenValue) + maxGasAmount),
+                        })
+                    }
 
-                return { type: 'deposit', response: prepareDepositTxsResponse, linkDetails, password, feeOptions }
+                    return { type: 'deposit', response: prepareDepositTxsResponse, linkDetails, password, feeOptions }
+                }
+            } catch (error) {
+                throw error
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)